src/LiveComponent/doc/index.rst

```diff
@@ -3104,34 +3104,40 @@ on the component:
 
 .. code-block:: diff
 
-    // ...
+      // src/Components/ProductSearch.php
+      namespace App\Components;
 
-    #[AsLiveComponent]
-    class ProductSearch
-    {
-      - #[LiveProp(writable: true)]
-      + #[LiveProp(writable: true, onUpdated: 'onQueryUpdated')]
-        public string $query = '';
+      use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+      use Symfony\UX\LiveComponent\Attribute\LiveProp;
 
-        // ...
+      #[AsLiveComponent]
+      class ProductSearch
+      {
+    -     #[LiveProp(writable: true)]
+    +     #[LiveProp(writable: true, onUpdated: 'onQueryUpdated')]
+          public string $query = '';
 
-        public function onQueryUpdated($previousValue): void
-        {
-            // $this->query already contains a new value
-            // and its previous value is passed as an argument
-        }
-    }
-}
+          // ...
 
-As soon as the `query` LiveProp is updated, the ``onQueryUpdated()`` method
+          public function onQueryUpdated($previousValue): void
+          {
+              // $this->query already contains a new value
+              // and its previous value is passed as an argument
+          }
+      }
+
+As soon as the ``query`` LiveProp is updated, the ``onQueryUpdated()`` method
 will be called. The previous value is passed there as the first argument.
 
 If you're allowing object properties to be writable, you can also listen to
-the change of one specific key:
+the change of one specific key::
 
-.. code-block::
+    // src/Components/EditPost.php
+    namespace App\Components;
 
-    // ...
+    use App\Entity\Post;
+    use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+    use Symfony\UX\LiveComponent\Attribute\LiveProp;
```

Comment:
Same here: you could keep the one for `Post` to help show how it is an entity, but these 2 use statements look unnecessary to me. WDYT?